src/Engine/Renderer/RenderObject/RenderObjectManager.hpp

```diff
@@ -36,6 +36,7 @@ class RA_ENGINE_API RenderObjectManager final
 
     /**
      * @brief Get all render objects, the vector is assumed to be empty when called
+     * \todo what does the previous sentence means ....
```

Comment:
Should be some remains from the old version which took the list as output parameter.
Can you update the doc?